src/host/CMakeLists.txt

```diff
@@ -15,6 +15,8 @@ add_local_sources(tb_common
 	file.c
 	ipc.c
 	schedule.c
+	edf_schedule.c
+	low_latency_schedule.c
 	topology.c
```

Comment:
For consistency I would go with ll_schedule.c

Reply:
Done